src/index.js

```diff
@@ -0,0 +1,9 @@
+
+function submitVal() {
+	event.preventDefault();
+	var val = document.getElementById("input1").value;
+	sessionStorage.setItem("val", val);
```

Comment:
Nice, that you used `sessionStorage` here, I am sure you would have been good to go to implement localStorage to keep the data when refreshing 👍🏽